components/tikv_util/src/logger/file_log.rs

```diff
@@ -115,7 +115,7 @@ impl Write for RotatingFileLogger {
                 fs::rename(&self.path, &new_path)?;
                 self.file = open_log_file(&self.path)?;
 
-                // Updates all rotators' states.
+                // Updates all roators' states.
```

Comment:
Please fix this typo.

Reply:
I intend to resubmit the original PR which includes this typo fix once I deal with the performance issue